src/vs/vscode.d.ts

```diff
@@ -1896,6 +1896,13 @@ declare module 'vscode' {
 		resolveCodeLens?(codeLens: CodeLens, token: CancellationToken): ProviderResult<CodeLens>;
 	}
 
+	export class DefinitionAndSpan {
```

Comment:
Also, I'm thinking of renaming this type. Something like `DefinitionAndDefiningSpan` or `BoundedDefinition` perhaps